benchmarks/perf-tool/README.md

```diff
@@ -311,6 +311,23 @@ Runs a set of queries with filter against an index.
 | recall@R | ratio of top R results from the ground truth neighbors that are in the K results returned by the plugin | float 0.0-1.0 |
 | recall@K | ratio of results returned that were ground truth nearest neighbors  | float 0.0-1.0 |
 
+#### get_stats
+
+Gets the index stats.
+
+##### Parameters
+
+| Parameter Name | Description                                                                                                                                                                                                                               | Default              |  
+| ----------- |-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------| 
+| index_name | Name of index to search                                                                                                                                                                                                                   | No default           |
+
+##### Metrics
+
+| Metric Name | Description                                     | Unit       |  
+| ----------- |-------------------------------------------------|------------|
+| num_of_committed_segments | Total number of commited segments in the index  | integer >= 0 |
```

Comment:
Whats difference between committed segments and search segments?

Reply:
committed segments are synced to disk and uncommitted are in translog only. 
search segments are searchable meaning index got refreshed. if I'm not mistaken without refresh search will return empty result even after ingestion is completed.
normally if index got refreshed number of committed and search segments is same